docs/kyma/04-03-cluster-installation.md

```diff
@@ -9,6 +9,8 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
 
 ## Prerequisites
 
+>**CAUTION:** Kubernetes is [deprecating Docker](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/) as a container runtime after v1.20 and [we are working on compatibility](https://github.com/kyma-project/kyma/issues/10842) all of our components with the `containerD`. Be sure that the kyma is not installed on cluster with the `containerD` and the `self-sign certificate` at the same time. If yes, upgrade the cluster to use the `docker` instead of the `containerD`, [generate the tls certificate](https://kyma-project.io/docs/#installation-install-kyma-with-your-own-domain-generate-the-tls-certificate) or [set an external docker registry](https://kyma-project.io/docs/components/serverless/#tutorials-set-an-external-docker-registry).
```

Comment:
**CAUTION:** Kubernetes is [deprecating Docker](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/) as a container runtime after v1.20 in favor of  `containerD`.
Be aware that, due to some [differences in CA handling in containrD](https://github.com/containerd/containerd/issues/3071),  kyma's built-in docker registry will not work correctly on clusters running with self-signed tls certificate on top of  k8s installation where  containerD is used  as container runtime.  If that is your case you should either upgrade the cluster to use the `docker` instead of the `containerD`, [generate valid tls certificate for your kyma instance](https://kyma-project.io/docs/#installation-install-kyma-with-your-own-domain-generate-the-tls-certificate) or [configure an external docker registry](https://kyma-project.io/docs/components/serverless/#tutorials-set-an-external-docker-registry).

Reply:
😉 
> **CAUTION:** As of version 1.20, Kubernetes is [deprecating Docker](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/) as a container runtime in favor of containerd. Due to a different way in which containerd handles certificate authorities](https://github.com/containerd/containerd/issues/3071), Kyma's built-in Docker registry will not work correctly on clusters running with a self-signed TLS certificate on top of Kubernetes installation where containerd is used as a container runtime. If that is your case, either upgrade the cluster to use Docker instead of containerd, [generate a valid TLS certificate for your Kyma instance](#installation-install-kyma-with-your-own-domain-generate-the-tls-certificate) or [configure an external docker registry](/components/serverless/#tutorials-set-an-external-docker-registry).